apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -273,16 +335,132 @@ onDocumentReady(() => {
+                      ? !row.point_difference
+                        ? html`<i class="bi bi-check-square-fill" style="color: green;"></i>`
+                        : html`<span style="color:red;">${row.point_difference}</span>`
+                      : !row.rubric_difference.length
+                        ? html`<i class="bi bi-check-square-fill" style="color: green;"></i>`
+                        : row.rubric_difference.map(
+                            (item) =>
+                              html`<div>
+                                ${item.false_positive
+                                  ? html`<i class="bi bi-plus-square-fill" style="color: red;"></i>`
+                                  : html`<i
+                                      class="bi bi-dash-square-fill"
+                                      style="color: red;"
+                                    ></i>`}
+                                <span>${item.description}</span>
+                              </div>`,
+                          )}`.toString(),
+              filterData: 'func:rubricItemsList',
+              filterCustomSearch: (text: string, value: string) =>
+                value.toLowerCase().includes(html`<span>${text}</span>`.toString()),
+              sortable: true,
+            }
+          : null,
+      ].filter(Boolean),
     ],
+    customSort: (sortName: string, sortOrder: string, data: InstanceQuestionRow[]) => {
+      const order = sortOrder === 'desc' ? -1 : 1;
+      if (sortName === 'rubric_difference') {
+        data.sort(function (a, b) {
+          const a_diff = a['point_difference'] === null ? null : Math.abs(a['point_difference']);
+          const b_diff = b['point_difference'] === null ? null : Math.abs(b['point_difference']);
+          if (a_diff === null && b_diff === null) {
+            // Can't compare if both are null
+            return 0;
+          } else if (a_diff !== null && b_diff !== null) {
+            if (a_diff === b_diff) {
+              // Same difference: compare number of disagreeing rubrics
+              return (
+                ((a['rubric_difference'] ? a['rubric_difference'].length : 0) -
+                  (b['rubric_difference'] ? b['rubric_difference'].length : 0)) *
+                order
+              );
+            } else {
+              // Normal comparison using point difference
+              return (a_diff - b_diff) * order;
+            }
+          } else if (a_diff !== null) {
+            // Make b appear in the end regardless of sort order
+            return -1;
+          } else {
+            // Make a appear in the end regardless of sort order
+            return 1;
+          }
+        });
+      } else {
+        (data as any[]).sort(function (a, b) {
+          if (a[sortName] < b[sortName]) {
+            return order * -1;
+          } else if (a[sortName] > b[sortName]) {
+            return order;
+          } else {
+            return 0;
+          }
+        });
```

Comment:
Huh, I wish there were a way to have `bootstrap-table` just fall back to its own default sorting algorithm, but I don't see any such option :(